authentication/src/test/java/io/camunda/authentication/session/WebSessionMapperTest.java

```diff
@@ -31,6 +36,34 @@ void setUp() {
     webSessionMapper = new WebSessionMapper(webSessionAttributeConverter);
   }
 
+  @Test
+  void toPersistentSessionWithCamundaPrinciple() {
```

Comment:
```suggestion
  void toPersistentSessionWithCamundaPrincipal() {
```

🤡 In principle, this should be principal

Reply:
Damn it! 🙈